recipes/agate/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{%set name = "agate" %}
+{%set version = "1.5.5" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "2861da5ef44a442a2225f0945a8ddce67ba4653622bf55b22f793a7b7fd95265" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py>35]
```

Comment:
Should this be `py3k` or it does work with `python <3.5`? (We are not building anything `< 3.5` so it does not matter for `conda-forge`, but it would be nice to have the right skip statement here in case someone else use this recipe outside `conda-forge`.)